app/controllers/tweet_controller.rb

```diff
@@ -0,0 +1,83 @@
+class TweetController < ApplicationController
+
+  get '/tweets' do
+    if logged_in?
+      erb :'tweets/index'
+    else
+      redirect '/login'
+    end
+  end
+
+  get '/tweets/new' do
+    if logged_in?
+      erb :'tweets/new'
+    else
+      redirect '/login'
+    end
+  end
+
+  post '/tweets' do
+    user = current_user
+    tweet = Tweet.create(params)
+    tweet.user = user
```

Comment:
The tweet creation and user association can both be accomplished in the same step:

``` ruby
@tweet = Tweet.create(content: params[:content], user: current_user)
```

or alternatively

```
@tweet = current_user.tweets.create(content: params[:content])
```